src/main/java/io/github/darealturtywurty/ancientology/core/data/RecipeGenerator.java

```diff
@@ -1,18 +1,26 @@
 package io.github.darealturtywurty.ancientology.core.data;
 
-import java.util.function.Consumer;
-
+import com.google.common.collect.ImmutableList;
+import io.github.darealturtywurty.ancientology.core.init.BlockInit;
+import io.github.darealturtywurty.ancientology.core.init.ItemInit;
 import net.minecraft.data.DataGenerator;
 import net.minecraft.data.recipes.FinishedRecipe;
 import net.minecraft.data.recipes.RecipeProvider;
+import net.minecraft.world.level.ItemLike;
+
+import java.util.function.Consumer;
 
 public class RecipeGenerator extends RecipeProvider {
+
+    private static final ImmutableList<ItemLike> TIN_SMELTABLES = ImmutableList.of(BlockInit.DEEPSLATE_TIN_ORE.get(), ItemInit.RAW_TIN.get());
```

Comment:
No need for `#get`s since `BlockRegistryObject` and `ItemRegistryObject` _already implements_ `ItemLike`

```suggestion
    private static final ImmutableList<ItemLike> TIN_SMELTABLES = ImmutableList.of(BlockInit.DEEPSLATE_TIN_ORE, ItemInit.RAW_TIN);
```

Reply:
The `#get` was implemented before `BlockRegistryObject` and `ItemRegistryObject` we're implemented, Ill go over all files and make the changes